python/foglamp/services/core/api/audit.py

```diff
@@ -49,100 +50,83 @@ class Severity(IntEnum):
 
 
 async def create_audit_entry(request):
-    """ Create a new Audit entry
+    """ Creates a new Audit entry
 
-    :Example:
-        JSON data
-        POST /foglamp/audit
+    Args:
+        request: POST /foglamp/audit
 
         {
-                "source"   : "LocalMonitor",
-                "severity" : "WARNING",
-                "details"  : {
-                                message" : "Engine oil pressure low"
-                             }
+          "source"   : "LMTR", # 5 char max
+          "severity" : "WARNING",
+          "details"  : {
+                        "message" : "Engine oil pressure low"
+                      }
         }
-    : curl example call
 
-        curl -X POST -d '{"source":"LocalMonitor","severity":"FATAL","details":{ message":"Engine oil pressure low"}} http://localhost:8081/foglamp/audit
+    :Example:
 
-    : returned JSON data on success
-
-    {
-        "timestamp" : "2017-06-21T09:39:51.8949395",
-        "source"    : "LocalMonitor",
-        "severity"  : "WARNING",
-        "details"   : { 
-                        message" : "Engine oil pressure low"
-                      }
-    }
-
-    Note:
-         only 4 levels are supported as the current methods in AuditLogger class
-         0 = Success
-         1 = Failure
-         2 = Warning
-         4 = Info
-    """
+        curl -X POST -d '{"source":"LMTR","severity":"WARNING","details":{"message":"Engine oil pressure low"}}
+        http://localhost:8081/foglamp/audit
 
+    Returns:
+        json object representation of created audit entry
 
-    _SEVERITY_PARAM = "severity"
-    _SOURCE_PARAM = "source"
-    _DETAILS_PARAM = "details"
+        {
+          "timestamp" : "2017-06-21T09:39:51.8949395",
+          "source"    : "LMTR",
+          "severity"  : "WARNING",
+          "details"   : {
+                         "message" : "Engine oil pressure low"
+                        }
+        }
+    """
 
     try:
         return_error = False
-        err_msg = "Missing required parameter "
+        err_msg = "Missing required parameter"
 
-        # Get input data
-        json_data = await request.json()
+        payload = await request.json()
 
-        # Set items to store
-        severity = json_data.get(_SEVERITY_PARAM)
-        source = json_data.get(_SOURCE_PARAM)
-        details = json_data.get(_DETAILS_PARAM)
+        severity = payload.get("severity")
```

Comment:
I think this was better before rather than having repeated strings in the code.

Reply:
same I believe. 3 vars + each used at 2 places only;